packages/replayio/src/utils/recordings/readRecordingLogLines.ts

```diff
@@ -0,0 +1,11 @@
+import { readFileSync } from "fs-extra";
+import { recordingLogPath } from "./config";
+
+export function readRecordingLogLines() {
+  const rawText = readFileSync(recordingLogPath, "utf8");
+  return rawText
+    .replace(/\}\{/g, "}\n{")
```

Comment:
do u know when a new line might not be inserted in between rows when writing to this log file?

Reply:
Nope, but I've seen it happen several times. I suspect something the runtime is doing doesn't jive with what this CLI is doine.